extensions/amp-consent/0.1/amp-consent.js

```diff
@@ -425,7 +425,6 @@ export class AmpConsent extends AMP.BaseElement {
    */
   init_() {
     this.passSharedData_();
-    this.maybeSetDirtyBit_();
```

Comment:
nit: I know we've already turned on the experiment flag. But to be extra safe can we do 
```
if (experimentOn) {
  
} else {
  this.maybeSetDirtyBit_();
}
```

This is to prevent error if the experiment is rolled back. It will be fairly easy to remove the else block when we clean up experiment. Thanks.